<ui version="4.0" >
 <class>Par2Window</class>
 <widget class="QWidget" name="Par2Window" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="outputGB" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>165</y>
     <width>371</width>
     <height>277</height>
    </rect>
   </property>
   <property name="title" >
    <string>PAR2 Tool Output</string>
   </property>
   <widget class="QTextEdit" name="outputTE" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>19</y>
      <width>351</width>
      <height>213</height>
     </rect>
    </property>
    <property name="undoRedoEnabled" >
     <bool>false</bool>
    </property>
    <property name="lineWrapMode" >
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly" >
     <bool>true</bool>
    </property>
    <property name="acceptRichText" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="forceCancelBut" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>237</y>
      <width>146</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Force Cancel Operation</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="specifyGB" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>7</y>
     <width>371</width>
     <height>143</height>
    </rect>
   </property>
   <property name="title" >
    <string>Specify Operation</string>
   </property>
   <widget class="QLineEdit" name="fileLE" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>22</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseBut" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>19</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>26</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>File:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goBut" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>90</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>GO!</string>
    </property>
   </widget>
   <widget class="QLabel" name="redundancyLbl" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>98</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Redundancy %:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="redundancySB" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>95</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>5</number>
    </property>
   </widget>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>55</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QRadioButton" name="createRB" >
       <property name="text" >
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="verifyRB" >
       <property name="text" >
        <string>Verify</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="repairRB" >
       <property name="text" >
        <string>Repair</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
